js/meetup/data.js

```diff
@@ -1,4 +1,59 @@
 const meetups = [
+  {
+    episode: 20,
+    title: 'RoR Himalayan Meetup on Zoom',
+    date: '2022-11-08',
+    start_time: '07:00 pm',
+    end_time: '09:00 pm',
+    address: 'Remote Meetup',
+    hosted_by: {
+      name: 'Jonathan Clarke',
+      twitter_username: 'sean_o_cleirigh'
+    },
+    venue_sponsor: {},
+    sessions: [
+      {
+        id: 1,
+        title: 'Keeping your secrets safe',
+        description: null,
+        speaker: 'Avi Bhurtel',
+        github_username: 'binarygit',
+        gitlab_username: '',
+        external_avatar: 'https://avatars.githubusercontent.com/u/87677429?v=4',
+        designation: 'Junior Developer @ OpenFoodNetwork.org',
+        slides_link: null,
```

Comment:
Pls download pdf from the google slide https://docs.google.com/presentation/d/1u1uuYMYcIkDIHwCiUZeLuHbsrWfeQhiF9dxLSmHdhZU/edit#slide=id.g18846017a2e_2_54

Reply:
Added the slides